### ISSUE TITLE
[🐞Bug] The absence of quantization loss make VQVAE to VAE.

### ISSUE THREAD

**Issue by cool-xuan** [PKU-YuanGroup/Open-Sora-Plan#138]
When I try to train VQVAE on my own data, I find the loss for vqvae training is only reconstruction loss https://github.com/PKU-YuanGroup/Open-Sora-Plan/blob/fdc786bc8e52d6386fb32c833eba0b4db286ca7b/opensora/models/ae/videobase/vqvae/trainer_vqvae.py#L11-L19, without the codebook loss like VideoGPT:
https://github.com/PKU-YuanGroup/Open-Sora-Plan/blob/fdc786bc8e52d6386fb32c833eba0b4db286ca7b/opensora/models/ae/videobase/vqvae/videogpt/videogpt/vqvae.py#L64-L69

Are you removing the quantization loss on purpose or unintentionally? And maybe the pretrained weights are trained by the official VideoGPT repo?

**Comment by qqingzheng** [comment]
I apologize for the oversight I made while refactoring the code (corresponding to pull request [#70](https://github.com/PKU-YuanGroup/Open-Sora-Plan/pull/70)).
